### PR TITLE
Gamedata update after 2/2/2023 CSGO update

### PR DESCRIPTION
**Pull request by GAMMACASE**
This pr is WIP to track and test out things and might be appended with further fixes. If you know anything that was also updated, comment about that in here or commit it directly (if possible). Also please, keep all the discussions in the appropriate thread in the AM discord, thanks.

This pr is also a collaborative work of: databomb#6706, Digby ° ૩ °#4534, Poggu#5993, Yuera#5880, PixeL#1337, Keldra#1114 and many others from the AM discord.

### CODE REVIEW

**Comment by ddorab** [comment]
Please double check yourself as well but I think CheckWinLimit is \xA1\x2A\x2A\x2A\x2A\x55\x89\xE5\x56\x53\x3D\x2A\x2A\x2A\x2A\x0F\x84\x2A\x2A\x2A\x2A\x8B\x10\x8B\x52\x2A\x81\xFA\x2A\x2A\x2A\x2A\x0F\x85\x2A\x2A\x2A\x2A\x8B\x70\x2A

**Comment by ddorab** [comment]
SetModelClass \x55\x89\xE5\x56\x53\x83\xEC\x1C\x8B\x5D\x2A\x8B\x75\x2A\x8B\x03\x53\xFF\x90\x2A\x2A\x2A\x2A\x83\xC4\x10\x84\xC0 as well

**Comment by Poggicek** [comment]
> 

Incorrect

**Comment by ddorab** [comment]
was that not the signature you sent?

**Comment by ddorab** [comment]
oh sorry, I thought I updated mine

**Comment by Poggicek** [comment]
> SetModelClass \x55\x89\xE5\x56\x53\x83\xEC\x1C\x8B\x5D\x2A\x8B\x75\x2A\x8B\x03\x53\xFF\x90\x2A\x2A\x2A\x2A\x83\xC4\x10\x84\xC0 as well

Also not correct. Both signatures should be okay.

**Comment by sev4a** [comment]
"Signatures"
		{
			"CCSGameRules::RewardMatchEndDrops"
			{
				"library"		"server"
				"linux"			"\x55\x89\xE5\x83\xEC\x58\xA1\x2A\x2A\x2A\x2A\x89\x5D\xF4\x89\x75\xF8\x0F\xB6\x5D\x0C"
				"windows"		"\x55\x8B\xEC\x83\xE4\xF8\xA1\x2A\x2A\x2A\x2A\x83\xEC\x1C\xB9"
			}

**Comment by ddorab** [comment]
am I looking at the wrong binary :/

**Comment by Poggicek** [comment]
> "Signatures"
{
"CCSGameRules::RewardMatchEndDrops"
{
"library" "server"
"linux" "\x55\x89\xE5\x83\xEC\x58\xA1\x2A\x2A\x2A\x2A\x89\x5D\xF4\x89\x75\xF8\x0F\xB6\x5D\x0C"
"windows" "\x55\x8B\xEC\x83\xE4\xF8\xA1\x2A\x2A\x2A\x2A\x83\xEC\x1C\xB9"
}

Sourcemod doesn't use this signature. PR to any existing repos using this function instead.

**Comment by asherkin** [comment]
Please take the time to fully consider what you're sharing and be precise about what you're talking about. If you just spam comment as soon as you have a thought this is going to quickly become unreadable - there are no awards for speed here.

**Review by qennu — APPROVED** [review]

**Review by qennu — APPROVED** [review]

File: gamedata/core.games/engine.csgo.txt [hunk 1]
```diff
@@ -38,9 +38,9 @@
 			/* Offset into LevelShutdown */
 			"gEntList"
 			{
-				"linux"		"13"
-				"linux64"	"13"
-				"mac64"		"13"
+				"linux"		"14"
+				"linux64"	"14"
+				"mac64"		"14"
```
Comment (data-bomb) [inline gamedata/core.games/engine.csgo.txt:43]: osx64/server.dylib did not look like it changed gEntList offsets. it's possible that only "linux" needs updating here but this needs more eyes on it

Comment (GAMMACASE) [reply gamedata/core.games/engine.csgo.txt]: Reverted.

**Review by asherkin — APPROVED** [review]
Thanks for the work here everybody!

**Pull request merged**